### ISSUE TITLE
colon_assignment_spacing breaks with variable properties

### ISSUE THREAD

**Issue by SlexAxton** [clutchski/coffeelint#201]
With this config:

``` json
  {
    "colon_assignment_spacing": {
        "name": "colon_assignment_spacing",
        "level": "error",
        "spacing": {
            "left": 0,
            "right": 1
        }
    }
  }
```

and this code being linted:

``` coffeescript
varname = 10

x =
  b: varname
```

The output says that it doesn't conform, when it does, in fact, conform.

``` sh
  ✗ coffeelinttest.coffee
     ✗ #4: Colon assignment without proper spacing. Expect colon spacing left: 0, right: 1. Got left: 1, right: 2..
```

Additionally, those internal values are always wrong (1 and 2, vs 0 and 1), but it only actually fails for variable properties.

May try and patch this myself, but I don't know the code well and thought maybe this'd be a better place to start.

Thanks for the killer software.
